spec/omniauth/strategies/untappd_spec.rb

```diff
@@ -12,7 +12,7 @@
                                'first_name' => 'John',
                                'last_name' => 'Doe',
                                'user_name' => 'john_doe',
-                               'untappd_url' => 'http://untappd.com/user/john_doe',
+                               'untappd_url' => 'https://untappd.com/user/john_doe',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.